pandas/io/clipboard.py

```diff
@@ -74,6 +74,12 @@ def to_clipboard(obj, excel=None, sep=None, **kwargs):  # pragma: no cover
       - Windows:
       - OS X:
     """
+    encoding  = kwargs.get('encoding')
```

Comment:
you need to pop this (or explicity list it), as you are passing thru kwargs below